### ISSUE TITLE
Add visit_id support to affwp_add_referral()

### ISSUE THREAD

**Issue by DrewAPicture** [AffiliateWP/AffiliateWP#1934]
It is currently only possible to associate a visit ID with a new referral when using db-layer methods directly.

This works:
```php
affiliate_wp()->referrals->add( array(
    'affiliate_id' => 10,
    'visit_id' => 12,
) );
```
This currently doesn't work:
```php
affwp_add_referral( array(
    'affiliate_id' => 10,
    'visit_id' => 12,
) );
```

PR + tests coming up.

**Comment by DrewAPicture** [comment]
Added a PR with tests: #1935

**Comment by pippinsplugins** [comment]
PR looks good to me!

**Comment by DrewAPicture** [comment]
Merged #1935.
